crates/fullnode/src/runner.rs

```diff
@@ -420,6 +423,13 @@ where
                     self.task_manager.abort().await;
                     return Ok(());
                 }
+                // Use this when SIGINT is not accepted
+                // Get the process id of the node and run `kill -15 <process_id>`
+                _ = shutdown_stream.recv() => {
+                    info!("Shutting down");
+                    self.task_manager.abort().await;
+                    return Ok(());
```

Comment:
Refactor this into:
```rust
fn shutdown(&self) {
    info!("Shutting down");
    self.task_manager.abort().await;
}
```
and call it from both ctrl_c and shutdown stream handlers.